src/libs/kata-types/src/config/default.rs

```diff
@@ -59,6 +59,7 @@ pub const MAX_DRAGONBALL_VCPUS: u32 = 256;
 pub const MIN_DRAGONBALL_MEMORY_SIZE_MB: u32 = 64;
 // Default configuration for qemu
 pub const DEFAULT_QEMU_BINARY_PATH: &str = "/usr/bin/qemu-system-x86_64";
+pub const DEFAULT_QEMU_ROOTFS_TYPE: &str = "ext4";
```

Comment:
There'll be two default rootfs types as "ext4", the other one is `DEFAULT_CH_ROOTFS_TYPE`. To simplify this, I suggest merging them into a single public default rootfs type for cases where "ext4" is desired.

Reply:
yep, I will bring with up to @pmores and change the code. thanks!